backend/app/agents/devrel/github/services/github_mcp_server.py

```
@@ -0,0 +1,88 @@
+GITHUB_ORG = os.getenv("GITHUB_ORG")
+if not GITHUB_ORG:
+    logger.warning("GITHUB_ORG not set in .env — defaulting to manual owner input")
+
+github_service: Optional[GitHubMCPService] = None
+try:
+    token = os.getenv("GITHUB_TOKEN") or os.getenv("GH_TOKEN")
+    if not token:
+        logger.warning("GITHUB_TOKEN/GH_TOKEN not set; GitHub API calls may be rate-limited or fail.")
+    github_service = GitHubMCPService(token=token)
+    logger.info("GitHub service initialized successfully")
+except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)